state.tf

```diff
@@ -1,15 +0,0 @@
-terraform {
```

Comment:
this file shouldn't be in modules, i.e. this one doesn't have it:
https://github.com/hmcts/cnp-module-postgres

it prevents testing locally with no state storage unless you remove it / comment it out